### PR TITLE
#M14. Misc cleanup changes and improvements

### PR DESCRIPTION
**Pull request by stefdworschak**
## Description
- Adding extra status between `registration_open` and `hack_in_progress` during which time the teams can be distributed, but will not be displayed yet.
- Adding assigned team to the hackathon view page for participants and mentors

## Pull request type

- [x] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue
#131

## Screenshots
image

### CODE REVIEW

**Comment by stefdworschak** [comment]
@TravelTimN another question is, should we even show all teams during a Hackathon to the participants as well?

File: hackathon/templates/hackathon/includes/judge_team_display.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            </li>
+        {% endfor %}
+        </ul>   
+        {% else %}
+        You are currently not assigned as a mentor.
+        {% endif %}
+        {% endwith %}
+    
+</div>
+{% endif %}
```
Comment (TravelTimN) [inline hackathon/templates/hackathon/includes/judge_team_display.html:24]: Missing new line.

**Review by TravelTimN — APPROVED** [review]
LGTM! One minor comment only (new line end of file missing)

**Comment by TravelTimN** [comment]
> @TravelTimN another question is, should we even show all teams during a Hackathon to the participants as well?

Yeah? Why not? We always show/announce the teams publicly anyway, so I think we can continue with this as per normal. Not sure why we'd hide the team information during a Hackathon?

**Comment by stefdworschak** [comment]
@TravelTimN yeah, no I am fine with sharing it. I just think it takes up a lot of real estate that we could maybe use to share some other content and we can maybe have a another tab with all the teams? Just a thought and nothing immediate.

File: hackathon/templates/hackathon/includes/judge_team_display.html (hunk 1 above)
Comment (stefdworschak) [reply hackathon/templates/hackathon/includes/judge_team_display.html]: Added a new line.

**Pull request merged**